[ Gardening ] Re-disabling RelaxThirdPartyCookieBlocking(api-test) as it's still failing.

Smackteo opened the pull request:
#### c2260113f19c152a7029ab61c093b5d843f586b8
<pre>
[ Gardening ] Re-disabling RelaxThirdPartyCookieBlocking(api-test) as it&apos;s still failing.
<a href="https://bugs.webkit.org/show_bug.cgi?id=280212">https://bugs.webkit.org/show_bug.cgi?id=280212</a>
<a href="https://rdar.apple.com/136531022">rdar://136531022</a>

Unreviewed test gardening.

*Tools/TestWebKitAPI/Tests/WebKitCocoa/Proxy.mm:

Canonical link: <a href="https://commits.webkit.org/287277@main">https://commits.webkit.org/287277@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/49ded7966e384b5fa2e2316a60d055b28cf64a7e

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/78913 "1 style error") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/57953 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/32294 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/83561 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/30155 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/81046 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/67089 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/6233 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/61786 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/19713 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/81980 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/51825 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/71795 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/42090 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/49175 "Passed tests") | [❌ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/25997 "Found 1 new test failure: imported/w3c/web-platform-tests/css/css-grid/subgrid/subgrid-baseline-008.html (failure)") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/28506 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/70289 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/26407 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/84940 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/6263 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/4323 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/70015 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/6426 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/67826 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/69266 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/13313 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/12049 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/12191 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/6211 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/6182 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/9624 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/7973 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/9a38568fe033f5fd5cbf2db640797811756a75a4. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/37336

webkit-commit-queue commented:
No reviewer information in commit message, blocking PR #37336. Details: [Build #12174](https://ews-build.webkit.org/#/builders/22/builds/12174)

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/49ded7966e384b5fa2e2316a60d055b28cf64a7e. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/37336

Smackteo committed:
[ Gardening ] Re-disabling RelaxThirdPartyCookieBlocking(api-test) as it's still failing.
https://bugs.webkit.org/show_bug.cgi?id=280212
rdar://136531022

Unreviewed test gardening.

*Tools/TestWebKitAPI/Tests/WebKitCocoa/Proxy.mm:

Canonical link: https://commits.webkit.org/287277@main

webkit-commit-queue commented:
Test gardening commit 287277@main (c2260113f19c): <https://commits.webkit.org/287277@main>

Reviewed commits have been landed. Closing PR #37336 and removing active labels.
